lib/fizzy/parser_expr.cpp

```diff
@@ -489,8 +489,46 @@ parser_result<Code> parse_expr(const uint8_t* pos, const uint8_t* end, const Mod
         case Instr::i64_store16:
         case Instr::i64_store32:
         {
+            size_t width = 0;
```

Comment:
This clearly not nice, should move it into `instructions.cpp`.

Reply:
@chfast what do you think, can we add a `bit_width`/`byte_width`/`memory_width` entry `InstructionMetrics`?